database/messages.js

```diff
@@ -1,69 +1,80 @@
 // connect to database
-require('./db');
+var db = require('./db');
 
 var mongoose = require('mongoose');
 var Promise = require('bluebird');
 mongoose.Promise = Promise;
 var messages = {};
 
 var MessageSchema = new mongoose.Schema({
+	id: Number,
```

Comment:
And not really critical but when I click "delete" on a message, I expect only that message to be erased.. I don't think it's correct to alter that behavior:
```
Guest115: LOL I love the new mariah carey!
haha so basic -Guest234
LOL I love the new mariah carey! -Guest234
LOL I love the new mariah carey! -Guest234
LOL I love the new mariah carey! -Guest234
see how dumb you sound -Guest234
```
^ should the first message be deleted? Extreme example but there it is